hack/release-tar

```diff
@@ -1,40 +1,49 @@
 #!/usr/bin/env bash
```

Comment:
We should keep the name of this script the same as buildx - so either we should follow-up and rename buildx's version to `release-tar`, or we should call this one `release`. No strong preferences either way.